data/data-pipeline/data_pipeline/score/score.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
```

Comment:
I don't know that this is a helpful parent class; furthermore, linting is not happy with me importing libraries here that I use only in child classes.

Reply:
Hmm, you do use it here though, in the typings? It's still not happy with that?

Reply:
should've put this comment on the following line - it's not happy with importing `FN`. @lucasmbrown-usds